azure-pipelines.yml

```diff
@@ -162,6 +162,56 @@ jobs:
               /p:TeamName=DotNetCore
               /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
             displayName: Sign packages
+    - job: Run_SDL
+      displayName: Run SDL tool
+      variables:
+        - group: SDL_Eng_KeyVault
+      dependsOn:
+        - Windows_NT
+        - Linux
+        - Validate_Helix
+        - Validate_Signing
+        - Asset_Registry_Publish
+      steps:
+        - checkout: self
+          clean: true
+        - powershell: eng\common\build.ps1
+            -configuration Release
+            -restore
+            -prepareMachine
+            -ci
+            /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
+          displayName: Windows Build / Publish
+        - task: NuGetToolInstaller@1
+          displayName: 'Install NuGet.exe'
+        - task: NuGetCommand@2
+          displayName: 'Install Guardian'
+          inputs:
+            restoreSolution: $(Build.SourcesDirectory)\eng\common\sdl\packages.config
+            feedsToUse: config
+            nugetConfigPath: $(Build.SourcesDirectory)\eng\common\sdl\NuGet.config
+            externalFeedCredentials: GuardianConnect
+            restoreDirectory: $(Build.SourcesDirectory)\.packages
+        # Script to execute SDL tools. TsaNotificationEmail will be changed to dnceng@ to check what the noise level would be with a regular SDL runs.
+        - powershell: eng/common/sdl/execute-all-sdl-tools.ps1
+            -GuardianPackageName Microsoft.Guardian.Cli.0.3.2
+            -NugetPackageDirectory $(Build.SourcesDirectory)\.packages
+            -Repository $(Build.Repository.Name)
+            -SourceDirectory $(Build.SourcesDirectory)
+            -ArtifactsDirectory $(Build.SourcesDirectory)\artifacts
+            -DncEngAccessToken $(dn-bot-dotnet-build-rw-code-rw)
+            -SourceToolsList @("policheck","credscan","apiscan","fxcop")
+            -ArtifactToolsList @("binskim")
+            -TsaInstanceURL "https://devdiv.visualstudio.com/"
+            -TsaProjectName "DEVDIV"
+            -TsaNotificationEmail "subal@microsoft.com"
```

Comment:
probs should be dotnetes@microsoft.com

Reply:
For now its my name to see how the noise traffic is for SDL running in a build. Then it will be switched. I have a comment already in place for this :)